components/Comment/CommentEditor.tsx

```diff
@@ -53,17 +59,40 @@ const CommentEditor = ({
     content,
   });
 
+  if (previewWhenInactive) {
+    useEffectHandleClick({
+      el: editorRef.current,
+      onOutsideClick: () => {
+        setIsPreviewMode(true);
+        isPreviewModeRef.current = true;
```

Comment:
Is this ref needed? I'm not seeing it used anywhere

Reply:
It is used. Let me huddle you and walk through this.